### ISSUE TITLE
[Task]: Update RunInference Notebooks to use Custom Inference Functions

### ISSUE THREAD

**Issue by jrmccluskey** [apache/beam#24334]
### What needs to happen?

Once Beam 2.44 is released, the RunInference example notebooks should be updated to take advantage of the custom inference function capabilities added in #22572

### Issue Priority

Priority: 2

### Issue Component

Component: run-inference

**Comment by ziqi-ma** [comment]
Would it be better to just add one more notebook using Huggingface generate rather than modifying the existing examples (since the existing ones don't really require calling a non-default inference function)?

**Comment by smeet07** [comment]
@jrmccluskey @damccorm I would like to contribute to this issue , I have one question though. What is the name of the convenience function that was agreed upon in #24062

**Comment by damccorm** [comment]
It depends on the model handler, but for PyTorchTensor its `make_tensor_model_fn` - https://github.com/apache/beam/blob/fbc7df425cd0cdd3e5e1dca05601f7b023dbed96/sdks/python/apache_beam/ml/inference/pytorch_inference.py#L154

Roughly they should all follow the `make_<something>_fn` pattern

**Comment by damccorm** [comment]
#27305 should address most of this, we still need to update https://github.com/apache/beam/blob/master/examples/notebooks/beam-ml/run_inference_pytorch_tensorflow_sklearn.ipynb

**Comment by smeet07** [comment]
@damccorm what model should I use for the example?

**Comment by damccorm** [comment]
Hey @smeet07 we shouldn't need to use a new model, we just need to get rid of `HuggingFaceStripBatchingWrapper` from https://github.com/apache/beam/blob/master/examples/notebooks/beam-ml/run_inference_pytorch_tensorflow_sklearn.ipynb and use a custom inference function instead

**Comment by smeet07** [comment]
Alright
